public/regdesk.html

```diff
@@ -53,23 +53,14 @@
         </div>
       </div>
     </nav>
-    <div class="row row-no-margin container-no-gutter">
-      <div id="buttonGutterLeft" class="col-2 d-grid gap-2 container-no-gutter">
-        <button id="cancelButton" class="btn btn-danger btn-fullheight">CANCEL</button>
+    <div class="row row-no-margin container-no-gutter h-100">
+      <div id="buttonGutterLeft" class="col-2 d-grid gap-2 h-100 container-no-gutter-left">
+        <button id="leftButton" class="btn btn-danger btn-fullheight btn-huge">CANCEL</button>
       </div>
-      <div class="col-8 pt-3">
-        <div class="row">
-          <div class="col-4 offset-4 d-grid gap-2 mb-3">
-            <button class="btn btn-primary" id="newSearch">🔎 New Search</button>
-          </div>
-        </div>
-        <div id="searchResults" class="row">
-          <!-- Demonstration canvas, actual layout will be different. -->
-          <canvas id="canvas" width="589" height="193" style="background-color: white; width: 589px; height: 193px; padding: 0;"></canvas>
-        </div>
+      <div class="col-8 pt-3" id="centerField">
       </div>
-      <div id="buttonGutterRight" class="col-2 d-grid gap-2 container-no-gutter">
-        <button id="printButton" class="btn btn-success btn-fullheight">PRINT</button>
+      <div id="buttonGutterRight" class="col-2 d-grid gap-2 h-100 container-no-gutter-right">
+        <button id="rightButton" class="btn btn-success btn-fullheight btn-huge">PRINT</button>
```

Comment:
I think this was more accurate when it was named printButton?

Reply:
Good point, rolled it back but with "btn" to match the CSS selector convention.